R/reportFE.R

```diff
@@ -802,6 +802,70 @@ reportFE <- function(gdx,regionSubsetList=NULL) {
                 setNames(tmp4[,,"FE (EJ/yr)"] - tmp4[,,"FE|+|Electricity (EJ/yr)"] - tmp4[,,"FE|+|Heat (EJ/yr)"], "FE|Fuels (EJ/yr)") 
   )
   out <- tmp5
+  
+  
+  ### FS: add new FE sectoral synfuel/biomass/fossil reporting
+  #( taking into account that the proportion of seliqfos and seliqbio flows may be different across sectors)
+  
+  ### disclaimer: (!) SE synfuels in REMIND normal are temporarily are part of segafos/seliqfos (this is differnet in REMIND-EU!)
+  
+  
+  # calculate synfuel share in SE seliqbio
+  
+  # only if CCU is on
+  if ("MeOH" %in% getNames(prodSE, dim=3)) {
+    vm_prodSE <- readGDX(gdx,name=c("vm_prodSe","v_seprod"),field="l",restore_zeros=FALSE,format="first_found")
+    
+    p_share_synfuel_liq <- collapseNames(vm_prodSE[,,"seliqfos.MeOH"] / dimSums(mselect(vm_prodSE, all_enty1="seliqfos"), dim=3))
+    p_share_synfuel_gas <- vm_prodSE[,,"h22ch4"] / dimSums(mselect(vm_prodSE, all_enty1="segafos"), dim=3)
+    
+    tmp8 <- NULL
+    
+    
+    tmp8 <- mbind(tmp8,
+                  setNames(p_share_synfuel_liq * collapseNames(prodFE[,,"seliqfos.fepet.tdfospet"]), 
+                           "FE|Transport|Liquids|LDV|Synthetic|New Reporting (EJ/yr)"),
+                  setNames(p_share_synfuel_liq * collapseNames(prodFE[,,"seliqfos.fedie.tdfosdie"]), 
```

Comment:
Ah and now I also remember why this is not so easy: There are `otherFEdemands` for `fedie` (which go to some CDR technology) which have to be subtracted first. Yeah I know.

Reply:
I tested all three gdx's and also w/o CCU. Should work. On the other FE diesel demand. I had not thought about this.  But I don't think it is a problem because, I looked it up, and only enhanced weathering requires diesel, which we will not switch on in the next months anyways, I guess. So that should be fine and we will have to redo the reporting anyways once the SE synfuel category is there.

Reply:
Okaaay. This will very likely be one of the "zombies that arise from the cellar". It is also quite annoying to not have a separate FE endpoint for this demand. Creates so much "boilerplate". Maybe mention the shortcoming in the comment.